styles.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: Roboto;
+  src: url(https://fonts.googleapis.com/css?family=Roboto);
+}
+
+@font-face {
+  font-family: Crete Round;
+  src: url(https://fonts.googleapis.com/css?family=Crete%20Round);
+}
+
+@font-face {
+  font-family: Poppins;
+  src: url(https://fonts.googleapis.com/css?family=Poppins);
+}
+
+body {
+  margin: -1px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+header {
+  display: flex;
+  background: #3c3a39;
+  color: #fff;
+  justify-content: space-between;
+  height: 48px;
+  align-items: center;
+  font-size: 18px;
+  width: 376px;
+}
+
+.logo {
+  color: #fff;
+  text-decoration: none;
+  width: 160px;
+  text-align: center;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+nav {
+  display: flex;
+  width: 68px;
+  justify-content: center;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  background: #1c1a19;
+  background-image: url(assets/Illustration.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+  color: #fff;
+  height: 590px;
+  justify-content: center;
+  width: 376px;
+  align-items: center;
+}
+
+.intro-text {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: flex-start;
+  order: 0;
+  width: 327px;
+}
+
+.primary-text {
+  font-size: 56px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+  color: #ff6b00;
+  width: 327px;
+  margin: -0.01px;
+  font-family: Crete Round, Arial, Helvetica, sans-serif;
+}
+
+.supporting-text {
+  width: 327px;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  text-align: left;
+  margin: -0.01px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.Description-text {
+  width: 330px;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+}
+
+.social-media {
+  display: flex;
+  justify-content: flex-start;
+  height: 50px;
+  align-items: center;
+  order: 1;
+  border: 5px solid rebeccapurple;
```

Comment:
- [ ] You have done an excellent job addressing the previous reviewer suggestions but there is an issue with the social icons because these are showing a border that is not in the Figma design. I would suggest removing this border to match the reference :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/143495833-4198690d-6d04-4548-9b78-b20a8686141b.png)